jrnl/output.py

```diff
@@ -25,17 +25,30 @@ def deprecated_cmd(old_cmd, new_cmd, callback=None, **kwargs):
         callback(**kwargs)
 
 
-def list_journals(configuration):
+def list_journals(configuration, format=None):
     from jrnl import config
 
     """List the journals specified in the configuration file"""
-    result = f"Journals defined in config ({config.get_config_path()})\n"
-    ml = min(max(len(k) for k in configuration["journals"]), 20)
-    for journal, cfg in configuration["journals"].items():
-        result += " * {:{}} -> {}\n".format(
-            journal, ml, cfg["journal"] if isinstance(cfg, dict) else cfg
-        )
-    return result
+    if format == "json":
+        import json
+
+        return json.dumps(configuration["journals"])
+    elif format == "yaml":
+        from io import StringIO
+
+        from ruamel.yaml import YAML
+
+        output = StringIO()
+        YAML().dump(configuration["journals"], output)
+        return output.getvalue()
+    else:
+        result = f"Journals defined in config ({config.get_config_path()})\n"
```

Comment:
Could you include this config path information in the other exports as well? I think `config_path` is a good key name for it.